run-gow

```diff
@@ -331,18 +341,18 @@ function get_compose_cmd() {
     echo "$cmd --project-directory \"${SCRIPT_DIR}\" --project-name gow $yaml_files"
 }
 
-function get_compose_version() {
-    local installed_version
+function get_compose_version {
+    local cmd="$(get_compose_exec_path)"
```

Comment:
this might cause issues with people who have a v1 `docker-compose` installed. v1's `version` output is in a different format, so it doesn't match the regex. it would end up saying "Docker Compose was not found" instead of a message that it's too old. that's not the end of the world, but it might be misleading.

Reply:
I think the version check would still work with v1 (unless it is a really old version of v1 that I have not tested)
I have installed v1 docker-compose and this is what I have:
```
┌─[josh5][mystique][~]
└─▪ compose_re='version v?([[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+)'
┌─[josh5][mystique][~]
└─▪ docker-compose version
docker-compose version 1.29.2, build unknown
docker-py version: <module 'docker.version' from '/home/josh5/.local/lib/python3.10/site-packages/docker/version.py'>
CPython version: 3.10.6
OpenSSL version: OpenSSL 3.0.2 15 Mar 2022
┌─[josh5][mystique][~]
└─▪ [[ "$(docker-compose version)" =~ $compose_re ]] && echo "${BASH_REMATCH[1]}"
1.29.2
┌─[josh5][mystique][~]
└─▪ docker compose version
Docker Compose version v2.11.2
┌─[josh5][mystique][~]
└─▪ [[ "$(docker compose version)" =~ $compose_re ]] && echo "${BASH_REMATCH[1]}"
2.11.2
```

Even though you are right in that these are different, the `v?` regex is zero or more, so it still parses fine.

Reply:
Hm, I could've sworn there was an issue with getting it to detect both back when I was originally working on it... I wonder what I'm remembering.  In any case, you're right, it should match just fine.